src/web/pages/WelcomePage.tsx

```diff
@@ -94,18 +104,30 @@ export default function WelcomePage({ navigate }: WelcomePageProps) {
     });
 
     (async () => {
-      const sessionState = (
+      const joinResult = (
         await joinSession({
           variables: {
             name,
             sessionId,
           },
         })
-      )?.data?.sessionState;
-
-      StorageUtil.local.setItem('sessionId', sessionId);
-
-      if (sessionState === SessionState.VOTING) {
+      )?.data?.join;
+
+      StorageUtil.session.setItem('sessionId', sessionId);
```

Comment:
The sessionId was stored in local storage so that it was persistent in the case of an accidental browser close; is that behavior maintained with this?

Reply:
@Xapphire13  That's a good question. As I was thinking about settings for the session, it made sense to me to store them in the `session` storage rather than `local` as it doesn't make sense for them to persist beyond a single use of the app. Once I had come to that conclusion it seemed logical to put all the session related data within `session` storage.
You're right that the accidental browser case is no longer handled, but I'm not sure that's such a problem. How do you feel about it?

Reply:
Yeah, I think you're right, the session stuff (including Id) probably make most sense the way you have it